.Lib9c.Plugin/Lib9c.Plugin.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <EnableDynamicLoading>true</EnableDynamicLoading>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Libplanet.RocksDBStore" Version="3.8.1" />
```

Comment:
It seems unnecessary.

```suggestion
```

Reply:
https://github.com/planetarium/lib9c/pull/2265/commits/11807c7ffa13208ee9fa31495a3012d3adf8103a amended!